visibility-sensor.js

```diff
@@ -9,6 +9,19 @@ if (typeof window !== 'undefined') {
   containmentPropType = React.PropTypes.instanceOf(window.Element);
 }
 
+function throttle (callback, limit) {
+    var wait = false;                 
+    return function () {              
+        if (!wait) {                  
+            wait = true;              
+            setTimeout(function () {
+                callback.call();
```

Comment:
Out of curiosity is there any reason for `callback.call()` instead of just `callback()` ?